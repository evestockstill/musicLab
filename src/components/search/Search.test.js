import React from 'react';
import { shallow } from 'enzyme';
import SearchBar from './Search';

describe('SearchBar component', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<SearchBar query='test query' onQueryChange={() => {}} onSubmit={() => {}}/>);
    expect(wrapper).toMatchSnapshot();
  });
});
