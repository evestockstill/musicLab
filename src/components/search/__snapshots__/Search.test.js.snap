// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar component matches a snapshot 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    type="text"
    value="test query"
  />
  <button
    className="button"
  >
    Search
  </button>
</form>
`;
